### PR TITLE
feat: [OIP-408]: Add Loki icon to Harness icons package

### PR DESCRIPTION
**Pull request by praneshg239**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/service-grafana-loki.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...="m8.02478 21.616-.23714-1.5923-1.59233.2372.27103 1.5923z" fill="url(#b)"/><path d="m20.4926 17.1441-.271-1.5584-6.9452 1.0841.2032 1.5585z" fill="url(#c)"/><path d="m11.0062 18.6347 1.5923-.2711-.2371-1.5584-1.5924.2371z" fill="url(#d)"/><path d="m10.5321 21.2096-.271-1.5585-1.55846.2372.20328 1.5923z" fill="url(#e)"/><path d="m5.82275 17.7877.23716 1.5923 1.59233-.2371-.23715-1.5924z" fill="url(#f)"/><path d="m13.6147 19.1088.2711 1.6263 6.9791-1.0842-.2372-1.5923z" fill="url(#g)"/><path d="m11.4126 21.0739 1.5585-.2033-.2372-1.6262-1.5923.271z" fill="url(#h)"/><path d="m8.2959 17.4149.271 1.5923 1.5585-.2371-.23722-1.5924z" fill="url(#i)"/><path d="m6.36474 16.8053-1.93112-12.67078-.71145.10163 1.96499 12.67085z" fill="url(#j)"/><path d="m7.27957 16.6697-2.10052-13.61941-.67759.13553 2.10052 13.58558z" fill="url(#k)"/><path d="m8.83824 16.4325-2.20216-14.4325-.67758.1355 2.20213 14.3648z" fill="url(#l)"/><path d="m9.75263 16.297-2.03275-13.14514-.64371.10163 2.03275 13.11121z" fill="url(#m)"/><path d="m11.3111 16.026-1.7956-11.65442-.67761.10163 1.79561 11.68839z" fill="url(#n)"/><path d="m12.226 15.8905-1.8634-12.09485-.67754.10164 1.89724 12.09491z" fill="url(#o)"/></g></svg>
```
Comment (AshwinBhatkal-Harness) [inline packages/icons/src/service-grafana-loki.svg:1]: nit: Do we require fill="none" here? If not, we can remove it

Comment (AshwinBhatkal-Harness) [reply packages/icons/src/service-grafana-loki.svg:1]: This also has linear gradient. Can we consider png instead?
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Comment (praneshg239) [reply packages/icons/src/service-grafana-loki.svg:1]: @AshwinBhatkal-Harness I will remove `fill="none"` attribute. As the size of SVG is not exceeding 5K, shall we use SVG itself as suggested in the link?

**Review by tan-nhu — DISMISSED** [review]

File: packages/icons/src/service-grafana-loki.svg (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply packages/icons/src/service-grafana-loki.svg:1]: Yes. Works

**Review by karansaraswat19 — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**